options.go

```diff
@@ -42,17 +50,32 @@ func DialerOptions(opts ...DialerOption) DialerOption {
 	}
 }
 
-// WithCredentialsFile returns a DialerOption that specifies a service account or refresh token JSON credentials file to be used as the basis for authentication.
+// WithCredentialsFile returns a DialerOption that specifies a service account
+// or refresh token JSON credentials file to be used as the basis for
+// authentication.
 func WithCredentialsFile(filename string) DialerOption {
 	return func(d *dialerConfig) {
-		d.sqladminOpts = append(d.sqladminOpts, apiopt.WithCredentialsFile(filename))
+		b, err := ioutil.ReadFile(filename)
+		if err != nil {
+			d.err = err
```

Comment:
Should this be a config error?

Reply:
It probably should be. We won't have the instance connection name in this context, but that's probably OK.